tensorflow/examples/tutorials/mnist/mnist_rnn.py

```diff
@@ -0,0 +1,120 @@
+""" Recurrent Neural Network.
+A Recurrent Neural Network (LSTM) implementation example using TensorFlow library.
+This example is using the MNIST database of handwritten digits (http://yann.lecun.com/exdb/mnist/)
+Links:
+    [Long Short Term Memory](http://deeplearning.cs.cmu.edu/pdfs/Hochreiter97_lstm.pdf)
+    [MNIST Dataset](http://yann.lecun.com/exdb/mnist/).
+"""
+
+from __future__ import print_function
+
+import tensorflow as tf
+from tensorflow.contrib import rnn
+
+# Import MNIST data
```

Comment:
Punctuation for comments: here and below.

Reply:
Thanks, updated.